docs/sources/reference/components/otelcol/otelcol.receiver.awss3.md

```diff
@@ -0,0 +1,222 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.receiver.awss3/
+description: Learn about otelcol.receiver.awss3
+labels:
+  stage: experimental
+  products:
+    - oss
+title: otelcol.receiver.awss3
+---
+
+# `otelcol.receiver.awss3`
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`otelcol.receiver.awss3` receives traces stored in S3 by the [AWS S3 Exporter](./otelcol.exporter.awss3.md).
```

Comment:
This is not the only way to store them in S3, it would be more precise to say that they're stored in OTLP proto or OTLP json. (until we add encoding extensions, then it's just stored in S3.

Reply:
Also this says only traces, should support any telemetry.